config/serving-net-istio.yaml

```diff
@@ -32,6 +26,18 @@ config:
         version: "4.17"
       - onDemand: true
         version: "4.13"
+    release-v1.17:
+      konflux:
+        enabled: true
+      openShiftVersions:
+      - candidateRelease: true
+        onDemand: true
+        skipCron: true
+        version: "4.18"
```

Comment:
I guess we need 4.18 for 1.17?

Reply:
In the [matix](https://docs.google.com/spreadsheets/d/1HTxR37_MM03_JQImW-KmAFk1oEQ83fXzfGxdfWQSX6o/edit?gid=0#gid=0) we don't have defined anything yet.
But in SOs project.yaml, we say 4.18: https://github.com/openshift-knative/serverless-operator/blob/c1e7f28388da1343eacf7e2e386eeea4b50b1b95/olm-catalog/serverless-operator/project.yaml#L32

Reply:
Could you update this for the other components too? :grin: :upside_down_face:

Reply:
Sure :)